modules/ffm-plus/src/main/java/club/doki7/ffm/library/ILibraryLoader.java

```diff
@@ -0,0 +1,17 @@
+package club.doki7.ffm.library;
+
+public sealed interface ILibraryLoader permits
+        WindowsLibraryLoader,
+        DefaultLibraryLoader
+{
+    ISharedLibrary loadLibrary(String libName) throws UnsatisfiedLinkError;
+
+    default ISharedLibrary loadLibraryFile(String fileName) throws UnsatisfiedLinkError {
+        return loadLibrary(fileName);
```

Comment:
To what extent is this implementation is correct? I think it's not true for jni, but for ffm it might be true (I don't know about it well enough). If there are only 2 impls and one of them don't use this default impl, maybe we should just push it downwards?